modules/fish/prompt.nix

```diff
@@ -7,6 +7,7 @@ let
 in ''
   source ${theme}
 
+  # See https://github.com/tomyun/base16-fish/issues/7 for why this condition exists
```

Comment:
I think ideally the reasoning is written out in text here. What if the project eventually gets migrated off of Github or if Github goes down?

Reply:
> I think ideally the reasoning is written out in text here.

Do you mean we should inline the explanation rather than delegating to an external source? I also prefer this, but I don't particularly mind in this case.

> What if the project eventually gets migrated off of Github

Hopefully, they would kindly link to the new location.

> or if Github goes down?

In that case, NixOS might go down for a while :eyes:

Reply:
> Do you mean we should inline the explanation rather than delegating to an external source?

Yes.

> Hopefully, they would kindly link to the new location.

I highly doubt that would happen. Comments age like raw fish.

Anyway, just a suggestion. I probably won't make the effort to update a comment.

Reply:
If `base16-fish` disappears / is moved we would have to migrate the entire module to a different solution anyway since users would no longer be able to fetch the flake input